### DISCUSSION TITLE
In custom filter allow to generate list of words ONLY containing provided letters.

### DISCUSSION THREAD

**Discussion by Kusnierz-Maciej** [monkeytypegame/monkeytype#5970] [Ideas]
I want to practice a set of words that ONLY contain the letters provided. A use case for that is to, for example, test new switches or keycaps by replacing only a few keys on the keyboard and generating a list of words containing only those letters.
I know I can add a list of excluded letters that are all alphabet letters except the ones in the include. But it gets tedious the do it manually.

**Comment by Leonabcd123** [comment]
#7126
